apps/ai-image-generator/package.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "ai-image-generator",
+  "version": "0.1.0",
```

Comment:
do we care about matching these versions to start initially? Idk how much we care about these things since we don't even bother to update them 🤷